### DISCUSSION TITLE
Images and link not visible

### DISCUSSION THREAD

**Discussion by rhaber-tech** [heroui-inc/heroui#3108] [Help]
Hi friends ! I am trying to use Next ui in my vite project and have run into 2 issue so far. First one being that the images are not visible (I've tried using local images as well) and the Links are not showing up even though everything else appears to be working just fine. 
Also I would like to thank you for taking some time out to read my question and helping me out. It makes a ton of difference in my learning journey.

**Comment by wingkwong** [comment]
please share your code.

**Reply by rhaber-tech** [reply]
Hello! I just copy pasted card component from the Next/ui website

**Reply by wingkwong** [reply]
Please provide a minimal reproducible example for us to investigate.

**Comment by alphaxek** [comment]
Hi @rhaber-tech, have you imported your images?

Refer below:
```
import img from './img.png';

<img src={img} alt="image" />
```
